src/cljc/athens/common_events/schema.cljc

```diff
@@ -260,6 +263,41 @@
       :target-uid  string?]]]])
 
 
+(def datascript-drop-same
+  [:map
+   [:event/args
+    [:map
+     [:drag-target keyword?
```

Comment:
`:drag-target` accepts only 3 keywords, please create an enum for it.

Reply:
Actually `:drag-target` can accept only 2 values `:above` and `:below`. `:child` value is never passed to the events that are triggered by drag-and-drop it is only there to decide if `drop/child` event should be called or not